src/test/java/com/amazon/opendistroforelasticsearch/security/dlic/rest/api/UserApiTest.java

```diff
@@ -448,7 +449,10 @@ public void testPasswordRules() throws Exception {
         Assert.assertTrue(response.getBody().contains("xxx"));
 
         response = rh.executePutRequest("/_opendistro/_security/api/internalusers/ok1", "{\"backend_roles\":[\"my-backend-role\"],\"attributes\":{},\"password\":\"\"}", new Header[0]);
-        Assert.assertEquals(HttpStatus.SC_OK, response.getStatusCode());
+        Assert.assertEquals(HttpStatus.SC_BAD_REQUEST, response.getStatusCode());
+
+        response = rh.executePutRequest("/_opendistro/_security/api/internalusers/non-empty_password", "{\"backend_roles\":[\"my-backend-role\"],\"attributes\":{},\"password\":\"Admin_123\"}", new Header[0]);
```

Comment:
The request creates new user. Please change it to update existing user "ok1".

Reply:
This intends to test user creation. Line 451-452 is to test updating existing user.

Reply:
User creation is tested using `addUserWithPassword()`, how this test is different?

Reply:
I change it to update the existing user "ok1".